release/src/main/groovy/MoblieGamingJavaUtils.groovy

```diff
@@ -0,0 +1,173 @@
+            "Beige",
+            "Bisque",
+            "BarnRed",
+            "BattleshipGrey"))
+
+    public static String generateCommand(String exampleName, String runner, TestScripts t){
+        if(exampleName.equals("UserScore")){
+            return generateUserScoreCommand(runner, t)
+        }
+        if(exampleName.equals("HourlyTeamScore")){
+            return generateHourlyTeamScoreCommand(runner, t)
+        }
+        if(exampleName.equals("LeaderBoard")){
+            return generateLeaderBoardCommand(runner, t)
+        }
+        if(exampleName.equals("GameStats")){
+            return generateGameStatsCommand(runner, t)
+        }
+        if(exampleName.equals("Injector")){
+            return generateInjectorCommand(t)
+        }
+        return "ERROR: Not found the example ${exampleName}."
+    }
+
+
+    private static String generateUserScoreCommand(String runner, TestScripts t){
+        StringBuilder cmd = new StringBuilder()
+        StringBuilder exeArgs = new StringBuilder()
+
+        exeArgs.append("--tempLocation=gs://${t.gcsBucket()}/tmp ")
+                .append("--runner=${runner} ")
+                .append("--input=gs://${t.gcsBucket()}/5000_gaming_data.csv ")
+        if(runner == "DataflowRunner"){
+            exeArgs.append("--project=${t.gcpProject()} ")
+                    .append("--output=gs://${t.gcsBucket()}/java-userscore-result-${RUNNERS[runner]}.txt ")
+        }
+        else{
+            exeArgs.append("--output=java-userscore-result-${RUNNERS[runner]}.txt ")
+        }
+
+        cmd.append("mvn compile exec:java -q ")
+                .append("-Dexec.mainClass=org.apache.beam.examples.complete.game.UserScore ")
+                .append("-Dexec.args=\"${exeArgs.toString()}\" ")
+                .append("-P${RUNNERS[runner]}")
+        return cmd.toString()
+    }
+
+
+    private static String generateHourlyTeamScoreCommand(String runner, TestScripts t){
+        StringBuilder cmd = new StringBuilder()
+        StringBuilder exeArgs = new StringBuilder()
+
+        exeArgs.append("--tempLocation=gs://${t.gcsBucket()}/tmp ")
+                .append("--runner=${runner} ")
+                .append("--input=gs://${t.gcsBucket()}/5000_gaming_data.csv ")
+        if(runner == "DataflowRunner"){
+            exeArgs.append("--project=${t.gcpProject()} ")
+                    .append("--output=gs://${t.gcsBucket()}/java-hourlyteamscore-result-${RUNNERS[runner]}.txt ")
+
+        }
+        else{
+            exeArgs.append("--output=java-hourlyteamscore-result-${RUNNERS[runner]}.txt ")
+        }
+
+        cmd.append("mvn compile exec:java -q ")
+                .append("-Dexec.mainClass=org.apache.beam.examples.complete.game.HourlyTeamScore ")
+                .append("-Dexec.args=\"${exeArgs.toString()}\" ")
+                .append("-P${RUNNERS[runner]}")
+        return cmd.toString()
+    }
+
+
+    private static String generateLeaderBoardCommand(String runner, TestScripts t){
```

Comment:
Is there some subset of default args that can be populated in advance that we're _sure_ we'll need for all of the xamples (runner, project, etc)